src/moh_civ_uat/AndroidManifest.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    package="org.medicmobile.webapp.mobile">
```

Comment:
```suggestion
<manifest xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:tools="http://schemas.android.com/tools">
```
For example look at [moh-togo-uat](https://github.com/medic/cht-android/blob/master/src/moh_togo_uat/AndroidManifest.xml)